File: Kattis/finding_an_a.py
# https://open.kattis.com/problems/findingana

from sys import stdin

def main():
    s = next(stdin).lower().strip()
    for i, l in enumerate(s):
        if l == "a":
            start = i
            break
    print(s[start:])

if __name__ == "__main__":
    main()

File: Kattis/finding_an_A.java
// https://open.kattis.com/problems/findingana

package Kattis;
import java.util.Scanner;

public class finding_an_A {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);
        String word = scanner.nextLine();
        scanner.close();
        String output = "";
        Boolean include = false;
        for (int i = 0; i < word.length(); i++) {
            if (word.charAt(i) == 'a') {
                include = true;
            }
            if (include) {
                output += word.charAt(i);
            }
        }
        System.out.println(output);
    }
}
